modules/user_manual/pages/files/public_link_shares.adoc

```diff
@@ -1,49 +1,47 @@
 = Public Link Shares
 
-With ownCloud X (10.0), we introduced the ability to create multiple
-public links per file or folder. This offers a lot of flexibility for
-creating different kinds of share links for a single file or folder,
-such as _different passwords_, _expiry dates_, and _permissions_.
+With ownCloud X (10.0), we introduced the ability to create multiple public links per file or folder. 
+This offers a lot of flexibility for creating different kinds of share links for a single file or folder, such as _different passwords_, _expiry dates_, and _permissions_.
 
-As of ownCloud version 10.0.2 you can xref:files/webgui/sharing.adoc#creating-drop-folders[create Drop Folders], 
-where users can upload files to a central location, but not be able to change any existing ones, nor see other 
-files which already have been uploaded.
+As of ownCloud version 10.0.2 you can xref:files/webgui/sharing.adoc#creating-drop-folders[create Drop Folders], where users can upload files to a central location, but not be able to change any existing ones, nor see other files which already have been uploaded.
 
-[[creating-public-link-shares]]
 == Creating Public Link Shares
 
-To create a public link share, first view the Sharing Panel of the file
-or folder that you want to create a public link share for, and under
-*"Public Links"* click *"Create public link"*.
+To create a public link share, first view the Sharing Panel of the file or folder that you want to create a public link share for, and under *"Public Links"* click *"Create public link"*.
```

Comment:
Why not using `menu:` and `btn:` 
(needs `:experimental:`)

Reply:
Fine by me.